### ISSUE TITLE
Recommend adding Multispectral column

### ISSUE THREAD

**Issue by coreysnipes** [OpenDroneMap/ODMdata#21]
@smathermather opening this change for discussion prior to adding a column on the main table in README.md...

Some of the sample ODM datasets are multi-spectral, and I think it would be helpful to provide that info in the table.  If it sounds OK I will add a column between RTK and Notes, called Multispectral.

**Comment by smathermather** [comment]
Yes, so maybe the column should be more generic, and have values like BW, RGB, NIR GB, NIR RG, Multispectral.  Sort of thinking out loud. There may be better ways to code it.

**Comment by coreysnipes** [comment]
OK great.  This is an area I don't understand well, so I'll open a topic on the forum and see what people suggest.

**Comment by smathermather** [comment]
For sure. I feel unqualified to answer this as well. :D
